.changeset/quick-cows-notice.md

```diff
@@ -0,0 +1,5 @@
+---
+'@makeswift/runtime': patch
+---
+
+swap out framer motion for box animations
```

Comment:
Can we use a full sentence here? This will make its way to the changelog and release notes so we want to make this as informative as possible